CHANGELOG.md

```diff
@@ -5,8 +5,15 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+Notable changes to `alacritty_terminal` crate should be documented in its
+[CHANGELOG](./alacritty_terminal/CHANGELOG.md).
+
 ## 0.14.0-dev
 
+### Fixed
+
+- New window sometimes being treated as focused when it's not on Wayland
```

Comment:
```suggestion
- New window being treated as focused when it's not on Wayland
```

Reply:
This bug is really hard to trigger though, I'm not sure you can even face it, but ok.

Reply:
Yeah it's just kinda unnecessary. Because people who aren't running into the bug just don't care. How frequently a bug occurred isn't really relevant. The people that ran into it know it's fixed, the people that never ran into it don't care either way.